pkg/registry/registry_antrea.go

```diff
@@ -72,8 +72,10 @@ func loadAntreaRegistry() {
 	registerInfoElement(*entities.NewInfoElement("throughputFromDestinationNode", 148, 4, 56506, 8), 56506)
 	registerInfoElement(*entities.NewInfoElement("reverseThroughputFromSourceNode", 149, 4, 56506, 8), 56506)
 	registerInfoElement(*entities.NewInfoElement("reverseThroughputFromDestinationNode", 150, 4, 56506, 8), 56506)
-	registerInfoElement(*entities.NewInfoElement("flowEndSecondsFromSourceNode", 151, 14, 56506, 4), 56506)
-	registerInfoElement(*entities.NewInfoElement("flowEndSecondsFromDestinationNode", 152, 14, 56506, 4), 56506)
+	registerInfoElement(*entities.NewInfoElement("flowEndSecondsFromSourceNode", 151, 3, 56506, 4), 56506)
```

Comment:
Could you help take a look at this @heanlan? I can't remember the reason behind the decision to use `uint32` for `flowEndSecondsFromSource/DestNode` instead of `dateTimeSeconds`.

Reply:
They should keep unchanged as uint32. We are following the same datatype with flowEndSeconds. https://github.com/vmware/go-ipfix/blob/ec54b43babc78f1c5f5e468333d4ebd72419455a/pkg/registry/registry_IANA.go#L148

Reply:
This change should be reverted. The datatype should be change back to 14.

Reply:
Sorry for creating some confusions here. I just noticed there are some discrepancies in both Antrea and Go-ipfix code. `FlowEndSecondsFromSourceNode` and `FlowEndSecondsFromDestinationNode` should have the same datatype with `FlowEndSeconds`. And the datatype is `dateTimeSeconds`. @tushartathgur could you revert this change and also change the following lines to `dateTimeSeconds`
https://github.com/vmware/go-ipfix/blob/ec54b43babc78f1c5f5e468333d4ebd72419455a/pkg/registry/registry_antrea.csv#L53-L54

The antrea doc is wrong:
https://github.com/antrea-io/antrea/blob/4806be3c14e21a75dd3b8e876785fbc947439474/docs/network-flow-visibility.md?plain=1#L486-L487

Reply:
Sure, Creating a separate PR for this as this is not part of L7Visibility.
Though I am confused, the registry_antrea.go should create the elements according to their datatype, however for these two fields, it should not have generated DateTimeSeconds format if the datatype was unsigned32 !

```
registerInfoElement(*entities.NewInfoElement("flowEndSecondsFromSourceNode", 151, 14, 56506, 4), 56506)
registerInfoElement(*entities.NewInfoElement("flowEndSecondsFromDestinationNode", 152, 14, 56506, 4), 56506)
```

Reply:
The underlying structure of DateTimeSeconds is uint32: https://www.rfc-editor.org/rfc/rfc7011.html#section-6.1.7

Reply:
I mean the registry_antrea.go is autogenerated, it should not have changed the unsigned32 to dateTimeSeconds

Reply:
Let's keep these two lines of change, i.e. 14 -> 3, with this PR, as we want to correct the errors. As you mentioned, we can do the uint32 -> dateTimeSeconds changes in a separate PR if we would like to.